### ISSUE TITLE
[US - DEV] As MILES HARDIN, I want to be able to navigate back from the 'terms and conditions' page to the previous page I visited.

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#1269]
### Considerations:
Miles Hardin wants to be able to navigate back from the 'terms and conditions' page to the previous page he visited. 

Please refer to:
- UX designs in [Figma ](https://www.figma.com/file/hEDIHsBZYa6hmK8As5mcjy/%5BDL---IdeaSpace%5D-UX-designs?node-id=2645%3A5947&t=YzCPmKYI4hbrBbcZ-1 ) 

### Acceptance criteria:
- When Miles Hardin clicks on the 'back' button on the 'terms and conditions' page, he should be taken back to the previous page he visited.
- The URL of the previous page should be preserved.
- The previous page should be displayed in the same state as when Miles Hardin left it.
- Layout is web responsive and the functionality were tested thoroughly and works seamlessly on all devices and [target browsers ](https://docs.google.com/spreadsheets/d/1upYP1mvHkU3sRBg4gCeGG4P63tO5-whn6VKLMlr6qYI/edit?usp=sharing ): Chrome, Microsoft Edge, Safari, Firefox and Opera.

----------------------------------------------------
### Sub-tasks:
- [X] UXR
- [X] UX design
- [x] Development
- [x] QA
- [x] Bug: If use link from Summary table, back button navigates to previously clicked Section ("Age Eligibility", "Changes to Terms", etc)
----------------------------------------------------

![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/d8e7d151-3a12-4bd6-a1bd-749dff2d6803)

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/1269?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @katehirschman @MariliaDP @RETprojects

**Comment by JulieMass** [comment]
Hi @alicefate ,

As we discussed yesterday in our sprint planning, I’ve created this ticket for the back button for the terms and conditions. Please let me know if the description and the acceptance criteria of the ticket are OK and/or if I missed anything.

Thank you,

**Comment by dbradham** [comment]
@JulieMass This is blocked until #750 is completed per the acceptance criteria.

**Comment by JulieMass** [comment]
> @JulieMass This is blocked until #750 is completed per the acceptance criteria.

Yep, I'm moving it to the blocked column. thank you Dave!

**Comment by dbradham** [comment]
@chadcrotchett @JulieMass flagging this ticket because it is in blocked but assigned to a developer no longer with the team.

**Comment by chadcrotchett** [comment]
@dbradham @JulieMass does "use the browser to navigate to the previous page" even require a User Story? I would assume it's owned by the browser, not Dev Launchers.

Also, since the Blocking item is in the IceBox, I would suggest this go into the IceBox as well. Thoughts?

**Comment by dbradham** [comment]
@chadcrotchett @AlinaU1 @julianalobosque is this still pending ui/ux review? Per our discussion earlier I deployed to QA

**Comment by chadcrotchett** [comment]
- [ ] test1

**Comment by chadcrotchett** [comment]
@Ahsaask Trying out our bug process for the first time. I have created a new sub-task on the body of this issue.

Bug: If use link from Summary table, back button navigates to previously clicked Section ("Age Eligibility", "Changes to Terms", etc) instead of going back to the "Submit an Idea" page. 

@AlinaU1 We really just want this button to be a "Close" button since we popped it up ourselves strictly for read-only, right?

cc: @andyzhu7568 Here's my first attempt at an in-line bug (1. create sub-task in body; 2. comment with additional details; 3. hyperlink to spreadsheet if you want traceability)

**Comment by AlinaU1** [comment]
Yes, as the T&Cs opens in a separate tab and we want a user to easily return to the previous page, the back button will close the tab with T&Cs. Is there an issue with this flow?

**Comment by chadcrotchett** [comment]
As discussed in Oct 9 meeting, back button doesn't work correctly after using page Section links. Will re-design as a Close button.
